app/forms/locationOfGoods/AddressFormProvider.scala

```diff
@@ -25,7 +26,7 @@ import play.api.i18n.Messages
 
 import javax.inject.Inject
 
-class LocationOfGoodsAddressFormProvider @Inject() extends Mappings {
+class AddressFormProvider @Inject() extends Mappings {
```

Comment:
Is this class identical to `forms.AddressFormProvider`?

Reply:
Looks like the generic form provider has a name param to show something like "Enter name's address" but the location of goods form provider doesn't have that requirement